FEWD/Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,87 @@
+var humanScore = 0;
+var computerScore = 0;
+
+function compChoice() {
+  var computerChoice = Math.random();
+  // console.log(computerChoice);
+  if (computerChoice <= 0.3) {
+    return computerChoice = "Rock";
+    // console.log(computerChoice);
+  } else if (computerChoice > 0.3 && computerChoice <= 0.69) {
+    return computerChoice = "Scissors";
+    // console.log(computerChoice);
+  } else {
+    return computerChoice = "Paper";
+    // console.log(computerChoice);
+  }
+}
+
+function userChoiceRock(e) {
+  e.preventDefault();
```

Comment:
Buttons don't have any default behaviour you need to prevent, but should that have been the case you would've done it perfectly like this 👍